src/function/mtk.jl

```diff
@@ -35,11 +35,17 @@ function instantiate_function(f, x, adtype::AutoModelingToolkit, p, num_cons=0)
         hv = f.hv
     end
 
+    expr = ModelingToolkit.Symbolics.toexpr(equations(sys))
+
     if f.cons === nothing
         cons = nothing
+        cons_exprs = nothing
     else
         cons = (θ) -> f.cons(θ, p)
         cons_sys = ModelingToolkit.modelingtoolkitize(NonlinearProblem(f.cons, x, p))
+
+        cons_eqs = equations(cons_sys)
```

Comment:
@shashi if I do an `equations(cons_sys) .~ 0)`, does `toexpr` generate it as `= 0`, or do I need to substitute `~` to `=`?

Reply:
>does toexpr generate it as = 0

It does